spec/support/shared/examples/msf/db_manager/service.rb

```diff
@@ -8,4 +8,61 @@
   it { is_expected.to respond_to :find_or_create_service }
   it { is_expected.to respond_to :services }
   it { is_expected.to respond_to :report_service }
+
+  describe '#report_service', if: !ENV['REMOTE_DB'] do
```

Comment:
Looks like the API for the remote db needs more work for existing functionality to be tested; but will mark this as skipped for now: https://github.com/rapid7/metasploit-framework/issues/18439